src/app/component/quiz/quiz.component.html

```diff
@@ -1,4 +1,3 @@
-<app-header></app-header>
 <div
   id="carouselExampleControlsNoTouching"
```

Comment:
what is use for this id ? id="carouselExampleControlsNoTouching" 
class="carousel slide"
 I guess no need for this

Reply:
yes, i removed it as it is not affecting anything.